commands/slugs/download.js

```diff
@@ -25,6 +28,9 @@ module.exports = {
   description: 'downloads a slug to <APP_NAME>/slug.tar.gz and then extracts it',
```

Comment:
How about changing the description as well to reflect the possible option.	
```suggestion
  description: 'downloads a slug to <APP_NAME>/slug.tar.gz and then extracts it if `no-extract` option not provided.',
```